### PR TITLE
removes --release from publish help

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
we deprecated this a while ago, we can still make it behave the same (alias of wrangler publish) but remove it from the help text :)

before

```console
$ wrangler publish --help
wrangler-publish 
🆙  Publish your worker to the orange cloud

USAGE:
    wrangler publish [FLAGS] [OPTIONS]

FLAGS:
    -h, --help       Prints help information
        --release    [deprecated] alias of wrangler publish
        --verbose    toggle verbose output

OPTIONS:
    -e, --env <env>    environments to publish to
```

after

```console
$ wrangler publish --help
wrangler-publish 
🆙  Publish your worker to the orange cloud

USAGE:
    wrangler publish [FLAGS] [OPTIONS]

FLAGS:
    -h, --help       Prints help information
        --verbose    toggle verbose output

OPTIONS:
    -e, --env <env>    environments to publish to
```

### CODE REVIEW

File: LICENSE-MIT [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 Ashley Williams <ashley666ashley@gmail.com>
+Copyright (c) 2020 Cloudflare, Inc. <wrangler@cloudflare.com>
```
Comment (ashleymichal) [inline LICENSE-MIT:1]: now this PR does two things, idk how  licensing things  work

File: src/main.rs [hunk 2]
```diff
@@ -470,6 +470,7 @@ fn run() -> Result<(), failure::Error> {
                 )
                 .arg(
                     Arg::with_name("release")
```
Comment (ashleymichal) [inline src/main.rs:472]: if we deprecated it, should it just go away?

File: LICENSE-MIT (hunk 1 above)
Comment (EverlastingBugstopper) [reply LICENSE-MIT:1]: uhh woops

File: src/main.rs (hunk 2 above)
Comment (EverlastingBugstopper) [reply src/main.rs:472]: eh - doesnt really hurt to keep it around so the command doesnt fail in case folks have it in CI but if you want we can remove

Comment (ashleymichal) [reply src/main.rs:472]: if  it's going into a  point release, we should probably leave it. would be a shitty reason for your ci  to fail. maybe  it doesn't get pulled until 2.0? idk.

Comment (EverlastingBugstopper) [reply src/main.rs:472]: yeah, makes sense

**Pull request merged**